src/count.py

```diff
@@ -0,0 +1,11 @@
+"""Get number of vowles in string."""
+
+
+def get_count(input_str):
+    """Get number of vowles in string."""
+    num_vowels = 0
+    a = list(input_str)
```

Comment:
You actually don't need to convert the string to a list here. A string is iterable.